src/DiffEqTutorials.jl

```diff
@@ -57,11 +57,11 @@ function weave_folder(folder)
   end
 end
 
-function tutorial_footer(folder=nothing, file=nothing)
+function tutorial_footer(folder=nothing, file=nothing; remove_homedir=false)
```

Comment:
we should probably just default this to true.

Reply:
OK, I'll change that.

Reply:
all fixed